openmc/universe.py

```diff
@@ -639,14 +639,20 @@ class DAGMCUniverse(UniverseBase):
     filename : str
         Path to the DAGMC file used to represent this universe.
     auto_geom_ids : bool
-        Set IDs automatically on initialization (True) or report overlaps
-        in ID space between CSG and DAGMC (False)
+        Set IDs automatically on initialization (True) or report overlaps in ID
+        space between CSG and DAGMC (False)
     auto_mat_ids : bool
-        Set IDs automatically on initialization (True)  or report overlaps
-        in ID space between OpenMC and UWUW materials (False)
+        Set IDs automatically on initialization (True)  or report overlaps in ID
+        space between OpenMC and UWUW materials (False)
     bounding_box : 2-tuple of numpy.array
         Lower-left and upper-right coordinates of an axis-aligned bounding box
         of the universe.
+    n_cells : int
+        The number of cells in the DAGMC model. This is the number of cells at
+        runtime and accounts for the implicit complement volume if it isn't
+        present in the DAGMC file.
```

Comment:
```suggestion
        The number of cells in the DAGMC model. This is the number of cells at
        runtime. This counts the implicit complement as a cell whether or not
        it is included in the DAGMC file as a explicit entity.
```

I'm not sure this wording is any better but wanted to have a go

Reply:
I think this is helpful! I didn't include the last sentence in https://github.com/openmc-dev/openmc/pull/2238/commits/3a13b95e2d4a45913b020804e1d42d3358d72a3c, but used most of the third! What do you think?